xds/googledirectpath/googlec2p.go

```diff
@@ -182,6 +182,13 @@ func newNodeConfig(zone string, ipv6Capable bool) map[string]any {
 		"id":       fmt.Sprintf("C2P-%d", randInt()),
 		"locality": map[string]any{"zone": zone},
 	}
+	if envconfig.NewPickFirstEnabled {
+		// Enable dualstack endpoints in TD.
+		// TODO(apolcyn): remove IPv6 metadata server queries entirely after old pick first is removed.
```

Comment:
If there isn't already one, could you please file an issue (internal or on github) to track this work?  Thanks!